components/ui/buttons/overlay.styl

```diff
@@ -1,14 +1,23 @@
 .Button
   &--overlay
+    // Fit 2-up on block hover
+    extra-padding = 0.125em
+    padding (button-v-padding + extra-padding) (button-h-padding + extra-padding)
+    min-width (block_width / 2.25)
+    font-size font-size-base
     background-color colors-gray-translucent
     border none
 
+    & + &
```

Comment:
like when `.foo` is next to another `.foo`, apply this style to the rightmost `.foo`